iris-mpc/src/bin/server.rs

```diff
@@ -285,6 +286,24 @@ async fn receive_batch(
                                 batch_size.clamp(1, max_batch_size);
                             tracing::info!("Updating batch size to {}", batch_size);
                         }
+                        if config.luc_enabled {
+                            if config.luc_lookback_records > 0 {
+                                batch_query.or_rule_serial_ids.push(
+                                    ((store_len as u32 - config.luc_lookback_records as u32)
+                                        ..store_len as u32)
+                                        .collect::<Vec<u32>>(),
+                                );
+                            }
```

Comment:
as @eaypek-tfh points out, `store_len` is not updated. However we also don't have access here to `current_db_sizes` in the actor. I think i's probably simplest to just pass the `luc_lookback_records` as a parameter in the batch and push to the records in the actor?

Reply:
Going with this suggestion, will implement the fix. Ideally we have a way of passing both (a set of serial ids and a lookback) and then merge them